### PR TITLE
Add .NET 5 target benchmark

### LINKED ISSUE #98: Add .NET 5 target benchmark

### PR DESCRIPTION
**Pull request by Sagilio**
close #98

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Sagilio please fix CI:

image

**Comment by Sagilio** [comment]
> @Sagilio please fix CI:
> 
> image

Already fixed.

**Comment by hsluoyz** [comment]
@huazhikui @xcaptain please review.

File: .github/workflows/github-actions-benchmark.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
     - name: Run Benchmark
       run: |
         cd Casbin.Benchmark
-        dotnet run -c Release -f netcoreapp31 --filter **
+        dotnet run -c Release -f netcoreapp3.1 --filter **
```
Comment (xcaptain) [inline .github/workflows/github-actions-benchmark.yml:41]: I have net5 sdk installed and tested this benchmark script, but I got some errors(I didn't install dotnet core 3.1 sdk).
图片
I think you should build the benchmark project in net5 sdk and run it in net5 runtime.

**Review by xcaptain — COMMENTED** [review]
would like to hear some benchmark tips using different sdk

**Comment by Sagilio** [comment]
> hear

I'd to hear about it.

File: .github/workflows/github-actions-benchmark.yml (hunk 1 above)
Comment (Sagilio) [reply .github/workflows/github-actions-benchmark.yml:41]: The benchmark program (it only will help test, and is not the test program.) will generate some different SDK projects then build and run it.  The -f argument is only for building the benchmark program.
If `Casbin.Benchmark` .csproj file like this: 
```xml
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <ServerGarbageCollection>true</ServerGarbageCollection>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <IsPackable>false</IsPackable>
  </PropertyGroup>
```
we will do not need the -f argument.

**Comment by hsluoyz** [comment]
@huazhikui

**Pull request merged**